docs/Configuration.md

```diff
@@ -630,17 +630,17 @@ Default: `[]`
 
 The paths to modules that run some code to configure or set up the testing environment. Each setupFile will be run once per test file. Since every test runs in its own environment, these scripts will be executed in the testing environment immediately before executing the test code itself.
 
-It's also worth noting that `setupFiles` will execute _before_ [`setupTestFrameworkScriptFile`](#setuptestframeworkscriptfile-string).
+It's also worth noting that `setupFiles` will execute _before_ [`setupFilesAfterEnv`](#setupFilesAfterEnv-array).
 
-### `setupTestFrameworkScriptFile` [string]
+### `setupFilesAfterEnv` [array]
 
-Default: `undefined`
+Default: `[]`
 
-The path to a module that runs some code to configure or set up the testing framework before each test. Since [`setupFiles`](#setupfiles-array) executes before the test framework is installed in the environment, this script file presents you the opportunity of running some code immediately after the test framework has been installed in the environment.
+Multiple paths to modules that run some code to configure or set up the testing framework before each test. Since [`setupFiles`](#setupfiles-array) executes before the test framework is installed in the environment, this script file presents you the opportunity of running some code immediately after the test framework has been installed in the environment.
```

Comment:
suggestion: "Multiple" -> "The" so we get consistent wording with `setupFiles` (or the other way around if Multiple suites better.

I also think we should mention that we're deprecating `setupTestFrameworkScriptFile`.